modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -109,3 +115,74 @@ Trace:
 When no MIME type is set on the payload, the MIME type defaults to
 `application/java`, and the content is handled as a `String` instead of a
```

Comment:
Compare this "a `String`" use with the one noted below: same context, different formatting?